lib/ProductOpener/Tags.pm

```diff
@@ -1765,6 +1765,14 @@ sub gen_tags_hierarchy($$) {
 	return @sorted_list;
 }
 
+my %and = (
+	en => " and ",
+	de => " und ",
+	es => " y ",
+	fi => " ja ",
+	fr => " et ",
+	it => " e ",
```

Comment:
```suggestion
	it => " e ",
	da => " og ",
	nl => " en ",
	cs => " a ",
	fi => " ja ",
	pt => " e ",
```

Reply:
https://www.indifferentlanguages.com/words/and

Reply:
I'll add them, don't commit it.